CONTRIBUTING.md

```diff
@@ -65,3 +34,27 @@ The Contributor understands and agrees that DeepChem shall have the irrevocable
 
 
 DeepChem understands and agrees that Contributor retains copyright in its Contributions. Nothing in this Contributor Agreement shall be interpreted to prohibit Contributor from licensing its Contributions under different terms from the [The MIT License](https://opensource.org/licenses/MIT) or this Contributor Agreement.
+
+## DeepChem Technical Steering Committee
```

Comment:
This content moves to the bottom. This is because many users want to know how to contribute firstly.

Reply:
Makes sense!